BoxContentSDK/BoxContentSDK/Models/BOXItem.m

```diff
@@ -248,7 +248,9 @@ - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
                                                                         nullAllowed:YES];
```

Comment:
we're doing "nullAllowed" here, however we're not handling the scenario where we pull in an NSNull value instead of a valid array. Should we handle that? or "supressNullAsNil"?